app/latigo/metadata_api/metadata_exceptions.py

```diff
@@ -0,0 +1,7 @@
+"""Exceptions for the Metadata API logic."""
+
+
+class MetadataStoringError(Exception):
+    """Raise if got not 200 status code from the API."""
+
+    pass
```

Comment:
```suggestion
class MetadataStoringError(Exception):
    """Raise if got not 200 status code from the API."""
```

`pass` is redundant since there is already a docstring in the class body.

Reply:
deleted, this exception is not needed anymore